### ISSUE TITLE
First beta release

### ISSUE THREAD

**Issue by kroyee** [kroyee/SpeedBlocks#75]
Howdy people.

I talked to @chinatsu today, and the game-server on darn.site can stay for until probably end of may. I suggest that we change up the workflow a bit and do this from today onwards:
- Make a development branch on both server & client where we can push new things.
- Make a first beta build for Mac/Win/Linux and make it available to people under releases
- Let the server at darn.site use the master branch so the server version matches the releases
- Post on cultris forum/HD that they can download and try it

Before we do this I think we should merge in the curl branch that me and Noir- was working on, so the Bug Report in the game sends us actual report. :+1:
This will require and additional library to build, libcurl. The reason for this is to be able to use https:// witch we for security reason definitely want to have access to in the future as well.

That was all.

PS. I put a donation button that's visible at the bottom of the readme on the start-page. It says collecting money for VPS server solution and that it costs €50-100 per year.

**Comment by kroyee** [comment]
I made the release builds for Win and Mac. Tried yesterday and there was some obvious issue with the room creating, so I tracked that down and fixed it today.
Now there are 2 new builds, I will pin them on Discord for now and people can try it.
The builds are made from the curl branch (it is up-to-date with the master branch but contains the libcurl additions aswell).

If all seems ok I'll move on to putting them on the release page.

There was also some issue yesterday with my dll files. Still not really clear to me why, but it would be nice if we could post the release with as little extras as possible. That means all the -d.dll (debug) don't have to be in there. The same thing SHOULD go for the .lib files afaik.
If someone on a Windows machine could verify this would be great.
